cmd/server/main.go

```diff
@@ -46,7 +53,17 @@ func main() {
 		MaxAge:           12 * time.Hour,
 		AllowWildcard:    true,
 	}
+
+	api.InitializeLimiters()
+	log.Info().Msg("Rate limiters initialized")
+
+	router := gin.New()                          // empty engine
+	router.Use(gin.Recovery())                   // add recovery middleware
+	router.Use(api.CustomGinLogger(&log.Logger)) // add our custom gin logger
+
 	router.Use(cors.New(config))
+	router.Use(api.GenerousRateLimiter())
```

Comment:
we might need to remove this since the GenerousRateLimiter will be at the highest level, which means all the routes will be limited by this.